wiki/osu!tourney/ko.md

```diff
@@ -0,0 +1,344 @@
+    `scoremode` - 0: 스코어, 1: 정확도, 2: 콤보
+-   `!mp move` <username> `[<slot>]` - 플레이어를 방으로 이동시켜 지정된 슬롯에 넣습니다. 플레이어가 이미 방에 있으면 플레이어는 지정된 슬롯으로 이동합니다.
+-   `!mp host` <username> - 플레이어에게 방장을 전달합니다.
+-   `!mp clearhost` - 방장을 지웁니다.
+-   `!mp settings` - 전체 세부 정보를 표시합니다.
+-   `!mp start [<time>]` - 설정된 시간 (초) 후에 또는 시간이 없는 경우에는 즉시 시작합니다.
+-   `!mp abort` - 경기를 중단합니다.
+-   `!mp kick` <username> - 플레이어를 추방시킵니다.
+-   `!mp team` <username> <colour> - 플레이어를 지정된 팀으로 이동시킵니다.
+    `colour` - red, blue
+-   `!mp map` <mapid> `[<playmode>]` - 방의 비트맵과 플레이모드를 변경합니다.
+    `playmode` - 0: osu!, 1: osu!taiko, 2: osu!catch, 3: osu!mania
+-   `!mp mods` <mod> `[<mod>] [<mod>] …` - 현재 적용중인 모드를 모두 제거하고, 이 모드를 방에 적용합니다.
+    `mod` - HR, DT, FL, HD, FI, Freemod, None
+-   `!mp timer [<time>]` - 카운트다운 타이머를 시작합니다.
+    '시간'은 30 초입니다.
+    타이머 알림은 매 순간 30초, 30초, 10초, 5초미만으로 나타납니다.  
+-   `!mp aborttimer` - 현재 타이머를 중지합니다. (게임시작, 일반 타이머 모두 적용)
+-   `!mp addref <username>, <username>...` - 방에 심판을 추가하여 채팅을 볼 수 있고, mp 명령을 사용할 수 있습니다.
+-   `#`<userid>는 위의 모든 명령에서 <username>을 대체 할 수 있습니다. 사용자 이름에는 밑줄 (**_ **)로 대체 된 공백이 있어야 합니다.
+-   `!mp password` - 멀티 플레이어 방 암호화 활성화 / 비활성화 (초기화된 경우).
+
+다음은 명령 사용의 예입니다.
+
+-   `!mp move Zallius 1` - Zallius를 방의 슬롯 1로 이동시킵니다.
+-   `!mp move Loctav 4` - Loctav를 방의 슬롯 4로 이동시킵니다.
+-   `!mp team Zallius blue` -Zallius를 파란색 팀으로 이동시킵니다.
+-   `!mp team Loctav red` - Loctav를 빨간색 팀으로 이동시킵니다.
+-   `!mp set 0 2` - 개인전, 스코어 모드로 설정합니다.
+-   `!mp start` - 즉시 게임을 시작합니다.
+
+### 방 관리
+
+osu! tourney 클라이언트가 방을 성공적으로 인식하려면 !mp move 및 !mp team 명령을 사용하여 방에 들어오는 플레이어에게 올바른 슬롯으로 배정해야합니다.
+
+**osu!tourney 사용법** 섹션에서 설명한대로, 클라이언트의 각 화면은 멀티 플레이어 로비슬롯에 배정됩니다. 
+tournament.cfg의 TeamSize 변수 값에 따라 왼쪽 팀이 첫 번째 슬롯을 차지하고 오른쪽 팀이 아래 슬롯을 차지합니다.
+
+![osu!tourney 플레이어 배정](Osutourneyassignment.png "osu!tourney Player Assignment")
+
+예를 들어 TeamSize = 4로 설정하면 1, 2, 3, 4 번 슬롯은 왼쪽 팀에 속하게 되고, 5, 6, 7, 8 번 슬롯은 오른쪽 팀에 속하게 됩니다. 
+만약 당신이 'TeamSize = 3'을 설정하면 1, 2, 3 슬롯은 왼쪽 팀에 속하게 되고, 4, 5, 6 번 슬롯은 오른쪽 팀에 속할 것입니다.
+
+_각 멀티방에 해당하는 화면_
+
+**Tip : 왼쪽 팀은 항상 파랑 팀이고, 오른쪽 팀은 항상 빨강 팀입니다.**
+
+osu!tourney는 모든 팀의 색깔을 무시하는 것을 명심해주시기 바랍니다. 
+유일하게 주목할 만한 것은 멀티방의 위치입니다.
+
+모든 팀 멤버가 osu!tourney 내에 있는 팀에 속해 있도록 하려면 선수들은 반드시 올바른 슬롯에 위치해야 합니다. 아래 그림은 다양한 팀사이즈를 가진 osu!tourney 사진입니다. 화면의 숫자는 멀티 플레이어 로비에서 배정된 슬롯을 나타냅니다. 숫자는 보이지 않으며 설명 목적으로만 여기에 표시됩니다.
+
+![TeamSize = 4](Osutourneywindows.png "TeamSize = 4")
+
+_TeamSize = 4_
+
+---
+
+![TeamSize = 3](Teamsize3.png "TeamSize = 3")
+
+_TeamSize = 3_
+
+---
+
+![TeamSize = 2](Teamsize2.png "TeamSize = 2")
+
+_TeamSize = 2_
+
+---
+
+![TeamSize = 1](Teamsize1.png "TeamSize = 1")
+
+_TeamSize = 1_
+
+### 스킨
+--------
+
+![osu!tourney 다양한 방법으로 커스텀마이즈 할 수 있다.](Osutourneycustom.png "osu!tourney can be customized in various ways")
+
+osu! tourney는 다양한 디자인을 위해 클라이언트 스킨이 수정 가능 합니다. 
+그것을 위해서 당신은 토너먼트 폴더에 스킨을 넣어야 합니다.
+
+폴더의 경로는 `.../osu!/Skins/User/tournament`입니다.스킨 요소는 이 폴더(tournament)에 배치 할 수 있으며 `*.jpg`, `*.png` and `*.mp4` 파일 확장자를 지원합니다. 적절한 스킨 배치를 위하여 다음요소에 따라 이름을 지정하십시오.
```

Comment:
all references to mp4 and/or video are to be removed (osu!tourney doesn't use them anymore; see #145)